### PR TITLE
Add data generation script

### PR DESCRIPTION
**Pull request by timothydadams**
@JasperC1999 @jayrbarr

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "enzyme-adapter-react-16": "^1.15.2",
     "express": "^4.17.1",
     "faker": "^4.1.0",
+    "json2csv": "^5.0.1",
     "mongoose": "^5.9.11",
```
Comment (jayrbarr) [inline package.json]: Are you still using the json2csv module?

**Review by jayrbarr — APPROVED** [review]
lgtm

File: package.json (hunk 1 above)
Comment (timothydadams) [reply package.json]: nope, i'll remove it super quick

File: db/dataSMALL.csv [hunk 2]
```diff
@@ -0,0 +1,1273 @@
...
+ 
Iusto saepe quam fuga quis distinctio ullam quaerat et eligendi. Et consectetur et facilis dolor molestiae sunt tenetur. Voluptas et debitis.", 275, 492, Sat Feb 22 2020 22:11:12 GMT-0800 (Pacific Standard Time), 13, 1, "Ralph_Kuhlman77", false, false, 321, 17, "http://d1i5z9gkmthkca.cloudfront.net/photos/product/333.jpeg", "3 Years of Service", 753, 21, 41
+57, "Ergonomic Soft Towels", 3, 2, 50.8, "Repudiandae voluptas sed voluptatem voluptatem tenetur veniam voluptate libero. Ratione ducimus non recusandae aut reiciendis officiis. Ad et qui reiciendis. Sint vel inventore quam pariatur excepturi qui assumenda voluptatem dolor. Consequatur in ducimus. Incidunt ullam inventore accusantium quam non perferendis.
+ 
Voluptate odit qui dolorum facilis molestias iste. Totam id accusamus necessitatibus recusandae. Non et nostrum odit. Eos reiciendis vel et saepe mollitia qui quasi id amet.
+ 
Sapiente consequatur accusamus temporibus. Similique ea aut quos ut nemo et eligendi deserunt magni. V
```
Comment (JasperCDev) [inline db/dataSMALL.csv]: You might want to gitignore this

**Review by JasperCDev — APPROVED** [review]
lgtm!

**Pull request merged**